fesod-sheet/src/main/java/org/apache/fesod/sheet/support/ExcelTypeEnum.java

```diff
@@ -50,7 +50,12 @@ public enum ExcelTypeEnum {
     /**
      * xlsx
      */
-    XLSX(".xlsx", new byte[] {80, 75, 3, 4});
+    XLSX(".xlsx", new byte[] {80, 75, 3, 4}),
+
+    /**
+     * ods (OpenDocument Spreadsheet)
+     */
+    ODS(".ods", new byte[] {80, 75, 3, 4});
```

Comment:
as ODS has same magic number with XLSX, so any potential issue during the magic usage in project level?

Reply:
> as ODS has same magic number with XLSX, so any potential issue during the magic usage in project level?

Check the internal structure of ZIP to distinguish (e.g., XLSX has xl/workbook.xml, ODS has content.xml).